infra/simple_lambda_layer.py

```
@@ -153,11 +153,27 @@ def _get_buildspec(self):
             force_destroy=True,
+            tags={
+                "Name": f"simple-lambda-layer-{self.name}-artifacts-{pulumi.get_stack()}",
+                "Service": self.name,
+                "Environment": pulumi.get_stack(),
+                "ManagedBy": "Pulumi",
+            },
+            opts=pulumi.ResourceOptions(parent=self),
+        )
+
+        # Explicitly enable versioning as a separate resource and depend on it where needed
+        bucket_versioning = aws.s3.BucketVersioning(
```

Ruff (0.13.1)

Local variable `bucket_versioning` is assigned to but never used

Remove assignment to unused variable `bucket_versioning`

(F841)